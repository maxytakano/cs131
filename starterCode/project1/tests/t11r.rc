float af[10];
float aff[10][20];
int ai[10];

float x[10][20];
int y[10];
float z[20];

function : void foo(float &x[10][20], int &y[10][20], float &z[10][20]) {
	z[19][9] = x[0][0] + y[0][0] + z[0][0];
}

function : void main()
{
	foo(z, x, x);
}
