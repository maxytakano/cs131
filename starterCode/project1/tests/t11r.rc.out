Error, "/tmp/cs131_testrunner_nGTaux/file.rc", line 10: 
  Index value of 19 is outside legal range [0,10).
Error, "/tmp/cs131_testrunner_nGTaux/file.rc", line 15: 
  Argument of type float[20] not equivalent to reference parameter x, of type float[10][20].
Error, "/tmp/cs131_testrunner_nGTaux/file.rc", line 15: 
  Argument of type float[10][20] not equivalent to reference parameter y, of type int[10][20].
Compile: failure.
